lacework/errors.go

```diff
@@ -1,16 +1,14 @@
 package lacework
 
 import (
-	"errors"
 	"log"
+	"strings"
 
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 func notFound(err error) bool {
-	var e *resource.NotFoundError
-	return errors.As(err, &e)
+	return strings.Contains(err.Error(), "404")
```

Comment:
When I look at the error in the failing pipeline in customer demo I see the error is
```
[GET] https://customerdemo.lacework.net/api/v2/VulnerabilityExceptions/VULN_A47591B615EC330E0CDDEE28D177FDE5A937FC62553EE18B587                                                                                                                                                              
[404] Not found   
```
Will `err.Error()` be this entire error string above? If so, is there any danger that the VULN GUID could contain `404`?

Reply:
Not sure if it comes through exactly as above, but perhaps we could check that it contains `[404]`?